apps/site/next-data/providers/blogData.ts

```diff
@@ -1,10 +1,18 @@
 import { cache } from 'react';
 
-import generateBlogData from '@/next-data/generators/blogData.mjs';
 import { BLOG_POSTS_PER_PAGE } from '@/next.constants.mjs';
+import { blogData as rawBlogData } from '@/next.json.mjs';
 import type { BlogCategory, BlogPostsRSC } from '@/types';
 
-const { categories, posts } = await generateBlogData();
+const blogData = {
```

Comment:
Can you move this object directly to the `const { categories, posts } =`

Reply:
sure, done 🙂 👍